common/cah-model.cpp

```diff
@@ -93,7 +93,17 @@ bool cah_model::prompt_trigger_popup(ux_window& window, const std::string& error
                 {
                     auto sd = *itr;
                     global_calib_status = RS2_CALIBRATION_RETRY; // To indicate in progress state
-                    sd->s->set_option(RS2_OPTION_TRIGGER_CAMERA_ACCURACY_HEALTH, 1.0f);
+                    try
+                    {
+                        sd->s->set_option(RS2_OPTION_TRIGGER_CAMERA_ACCURACY_HEALTH, 1.0f);
```

Comment:
Please don't use 1.0f -- we have an enum now. Use RS2_CAH_TRIGGER_NOW

Reply:
Done